doc/pi_setup.md

```diff
@@ -0,0 +1,107 @@
+# Raspberry Pi 3 setup
+
+## Create the Rasbpian Boot Disk
+Use [Apple Pi Baker](https://www.tweaking4all.com/software/macosx-software/macosx-apple-pi-baker/) to create a boot disk from the [Raspbian Jessie Lite](https://downloads.raspberrypi.org/raspbian_lite_latest) image.
+
+## Configuring SSH
+
+Insert the SD card into your development machine and add a file named `ssh` to SD card's root directory.
+
+Plug the Pi into the ethernet and find its ip address.
+
+You can then ssh into the machine:
+
+```
+$ ssh pi@<ip address>
+```
+
+The default password for this user is `raspbian`.
+
+
+## Update/Upgrade the System
+```
+$ sudo apt-get update
+$ sudo apt-get upgrade
```

Comment:
Looks like you duplicate this down in system setup, so probably keep one or the other